internal/provider/resource_dashboard.go

```diff
@@ -269,6 +292,7 @@ func (r *DashboardResource) Update(ctx context.Context, req resource.UpdateReque
 		DashboardTemplate: plan.DashboardTemplate,
 		TemplateBindings:  plan.TemplateBindings,
 		DashboardContent:  jsontypes.NewNormalizedValue(updatedDashboard),
+		Timeframe:         types.StringValue(dashboard.Timeframe),
 		Group:             types.StringValue(dashboard.Group),
 		Name:              types.StringValue(dashboard.Name),
 		SchemaVersion:     types.StringValue(dashboard.SchemaVersion),
```

Comment:
Ditto previous comments on SchemaVersion and Group (both fields are referened in a few other places in this repo I haven't commented on too as not changed in this PR)

Reply:
I will do a cleanup of these based on our new API documentation. I have raised a JIRA for this

Reply:
> I will do a cleanup of these based on our new API documentation. I have raised a JIRA for this

Thanks @shaswot77